vllm/multimodal/registry.py

```diff
@@ -28,6 +42,9 @@ def __init__(
             plugins: Sequence[MultiModalPlugin] = DEFAULT_PLUGINS) -> None:
         self._plugins = {p.get_data_key(): p for p in plugins}
 
+        self._init_limits_per_plugin = {k: 0 for k in self._plugins}
```

Comment:
Shouldn't this be `k: 1` if default limit is 1 for all modalities?

Reply:
This init_limits_per_plugin is actually the placeholder value used for non-multimodal models. I'll add a comment to be clearer about this.